### PR TITLE
HTML base tag - reverse proxy

### PR DESCRIPTION
**Pull request by Boehmi-TB**

### CODE REVIEW

**Comment by EndyCZ** [comment]
I would maybe even link  to official documetation that describes the purpose of base tag. 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base

**Comment by Boehmi-TB** [comment]
> I would maybe even link to official documetation that describes the purpose of base tag. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base

Added with commit https://github.com/industrial-edge/hello-world/pull/12/commits/a20053c5f26f4b9561e740aef7c73f8650f966ca

**Review by EndyCZ — CHANGES_REQUESTED** [review]
sorry

File: docs/implementation.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## Reverse Proxy
+
+In the Industrial Edge App Publisher the user has to add a configuration to use the reverse proxy of the Industrial Edge Device. 
+
+```txt
+  Container Port: 80
+  Protocol: HTTP 
+  Service Name: hello-world
+  Rewrite Target: /
```
[the remark this answers is not in the document]
Comment (Boehmi-TB) [reply docs/implementation.md:35]: added in link to docu in [95334fa](https://github.com/industrial-edge/hello-world/pull/12/commits/95334fae6d5f443ee4398a437e43d512a8249673)

File: docs/installation.md [hunk 2]
```diff
@@ -33,26 +35,28 @@ Download or clone the repository source code to your workstation.
...
-  - Define the platform in the docker-compose file, simply uncomment the line mentioning the platform in the docker-compose file
-  - Use the `docker compose build` (replaces the older `docker-compose build`) command to build the docker image of the service which is specified in the docker-compose.yml file.
-  - These Docker images can now be used to build your app with the Industrial Edge App Publisher
-  - `docker inspect " Image name" ` can be used to check for the image
+```bash
+sudo docker run --privileged --rm tonistiigi/binfmt -- install arm64
+```
+
+Afterwards, you can proceed with the following:
+
```
[the remark this answers is not in the document]
Comment (Boehmi-TB) [reply docs/installation.md:70]: added output in [b17193b](https://github.com/industrial-edge/hello-world/pull/12/commits/b17193b09467d453c9700c7eb9e8810ec2873f6f)

File: docs/installation.md [hunk 3]
```diff
@@ -33,26 +35,28 @@ Download or clone the repository source code to your workstation.
...
-  ```bash
-  sudo docker run --privileged --rm tonistiigi/binfmt -- install arm64
-  ``` 
-  Afterwards, you can proceed with the following:
-  - Define the platform in the docker-compose file, simply uncomment the line mentioning the platform in the docker-compose file
-  - Use the `docker compose build` (replaces the older `docker-compose build`) command to build the docker image of the service which is specified in the docker-compose.yml file.
-  - These Docker images can now be used to build your app with the Industrial Edge App Publisher
-  - `docker inspect " Image name" ` can be used to check for the image
+```bash
+sudo docker run --privileged --rm tonistiigi/binfmt -- install arm64
```
[the remark this answers is not in the document]
Comment (Boehmi-TB) [reply docs/installation.md]: fixed command in [b17193b](https://github.com/industrial-edge/hello-world/pull/12/commits/b17193b09467d453c9700c7eb9e8810ec2873f6f)

File: docs/installation.md [hunk 4]
```diff
@@ -33,26 +35,28 @@ Download or clone the repository source code to your workstation.
...
+```
+
+Afterwards, you can proceed with the following:
+
+- Define the platform in the docker-compose file, simply uncomment the line mentioning the platform in the docker-compose file
+- Use the `docker compose build` (replaces the older `docker-compose build`) command to build the docker image of the service which is specified in the docker-compose.yml file.
+- These Docker images can now be used to build your app with the Industrial Edge App Publisher
+- `docker inspect " Image name"` can be used to check for the image
```
[the remark this answers is not in the document]
Comment (Boehmi-TB) [reply docs/installation.md:74]: Solution is to have two docker-compose, one for amd64 one for arm64

[the remark this answers is not in the document]
Comment (EndyCZ) [reply docs/installation.md:74]: ah, simple solution. What about if we then add a a file called docker-compose.arm64.yaml for this purpose?

[the remark this answers is not in the document]
Comment (Boehmi-TB) [reply docs/installation.md:73]: @EndyCZ 
[a1dc2fe](https://github.com/industrial-edge/hello-world/pull/12/commits/a1dc2fee76dea1275547b59dabee43710d448836) added docker compose for arm64 and updated documentation accordingly

**Review by EndyCZ — APPROVED** [review]
Looks goood.

**Pull request merged**